src/components/UserInfo.js

```diff
@@ -1,18 +1,25 @@
 export default class UserInfo {
-  constructor({nameSelector, descriptionSelector}) {
+  constructor({nameSelector, descriptionSelector, avatarSelector}) {
     this._nameElement = document.querySelector(nameSelector);
     this._descriptionElement = document.querySelector(descriptionSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     return {
-      userName: this._nameElement.textContent,
-      userDescription: this._descriptionElement.textContent
+      name: this._nameElement.textContent,
+      about: this._descriptionElement.textContent,
+      avatar: this._avatarElement.src,
     }
   }
 
   setUserInfo(userData) {
     this._nameElement.textContent = userData.name;
-    this._descriptionElement.textContent = userData.description;
+    this._descriptionElement.textContent = userData.about;
+    this._avatarElement.src = userData.avatar;
```

Comment:
Needs correction: there is no need to set the avatar here and you don't update it when updating the user info. When initializing the user info you can use both methods `setUserInfo` to update the name and the job and ` setUserAvatar` to update the avatar

Reply:
This has been corrected.